vscode/src/chat/chat-view/ChatManager.ts

```diff
@@ -134,6 +135,13 @@ export class ChatManager implements vscode.Disposable {
         await chatProvider.executeCustomCommand(title, type)
     }
 
+    public async editChatHistory(treeItem?: vscode.TreeItem): Promise<void> {
+        const chatID = treeItem?.id
```

Comment:
```suggestion
        const chatID = treeItem?.id
        const chatLabel = treeItem?.label as vscode.TreeItemLabel
```

If you pass the treeLabel to the function here, we can remove the `getTreeItemLabelByID` function from TreeViewProvider which does the same thing as getTreeItemByID. Wdyt?

Reply:
yeah, you are right that can be done. I will update this to pass the chat label.